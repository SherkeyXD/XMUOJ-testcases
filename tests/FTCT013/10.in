20
-WWWWWWWWWDLWLWDDWLL
D-DDDDWDDDLDDDDLLDDD
DD-DLDDDDDDDDDDDDDDL
DWD-DLDDWDDDDLDDDDDD
DDDL-DDLDDDDLDDDDDDD
WWDWD-LDDWDDLDDDDDDD
DDDDDD-WDDDDDDDLDDDD
DDDDDDD-DLDDLDDDDDDD
DDDDDLDD-DDLDDDDDDLD
DDDDDWLDD-DDDDDDDWDD
DDDLLDDDDL-DDDDDDDDD
DDLDDDDDDDD-DDDDDLDD
DDDDDDLDLDDD-WLWDDDD
LDWDDLDDDLDDW-DWWDDD
DDDDDDDDDDDDLD-DDDDD
DDWDDDDDLWDDLLD-DDDD
DDDWDDDWDDDLDDDW-DDW
DDLWDDDDDDDDWDDDD-DW
DDLDDDDDDDDWLDDDDL-L
DDDWWDDDDDDDWDDLDWD-
