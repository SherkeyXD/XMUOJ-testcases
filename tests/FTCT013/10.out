1 15 13 6 4 9 16 3 18 20 19 12 17 8 5 11 2 7 10 14
